app/assets/javascripts/oxalis/view/input_catcher.js

```diff
@@ -21,20 +21,29 @@ function ignoreContextMenu(event: SyntheticInputEvent<>) {
 
 // makes the input catcher a square and returns its position within the document
 // relative to the rendering canvas
-function makeInputCatcherQuadratic(inputCatcherDOM: HTMLElement): Rect {
+function makeInputCatcherQuadratic(inputCatcherDOM: HTMLElement, makeQuadratic: boolean): Rect {
```

Comment:
The function name and comment should be adjusted to more closely reflect what the function is doing now